### PR TITLE
add documentation for query timeout

### PR DESCRIPTION
**Pull request by ObiWahn**
I think https://www.arangodb.com/docs/stable/aql/invocation-with-arangosh.html#setting-options is not the best location for options. I think It is a bit too hidden.

### CODE REVIEW

File: 3.6/release-notes-new-features36.md [hunk 1]
```diff
@@ -132,7 +132,10 @@ into a set of equal-distance buckets.
 
 ### Miscellaneous AQL changes
 
-ArangoDB 3.6 provides a new AQL function `GEO_AREA` for area calculations.
+ArangoDB 3.6 provides the following new AQL functionality:
+
+- function `GEO_AREA` for area calculations
```
Comment (Simran-B) [inline 3.6/release-notes-new-features36.md]:
```suggestion
- a function `GEO_AREA()` for area calculations
```

File: 3.6/release-notes-new-features36.md [hunk 2]
```diff
@@ -132,7 +132,10 @@ into a set of equal-distance buckets.
...
+- query option `timeout` to restrict the execution time.
```
Comment (Simran-B) [inline 3.6/release-notes-new-features36.md]:
```suggestion
- a query option `timeout` to restrict the execution time.
```

**Comment by ObiWahn** [comment]
> I agree. How else can the query option be set? In the query itself, like `OPTIONS { timeout: 123 }`?
No. It can be only set via the options parameter of `_query`. Only some operations like `INSERT` have extra options in AQL.
> Does it expect a number in seconds?
yes

**Comment by Simran-B** [comment]
will be backported to 3.5.4: https://github.com/arangodb/arangodb/pull/10405/files

**Comment by Simran-B** [comment]
> Only some operations like `INSERT` have extra options in AQL.

The FOR operation supports OPTIONS for index hints, but RETURN does not support any OPTIONS. Maybe a good feature request?

**Comment by jsteemann** [comment]
Things such as a query timeout shouldn't be specified in per-operation `OPTIONS`, as there can be multiple operations in the same query.
For example
```
FOR doc IN view OPTIONS { timeout: 10 }
  UPDATE doc._key WITH { ... } IN collection OPTIONS { timeout: 5 }
  RETURN NEW OPTIONS { timeout: 30 }
```
So it will really be better to set the timeout outside of per-operation `OPTIONS`.

**Pull request merged**